packages/ibm-products/src/components/CreateTearsheet/preview-components/MultiStepTearsheet.js

```diff
@@ -209,6 +212,29 @@ export const MultiStepTearsheet = (
                 }
                 checked={shouldIncludeAdditionalStep}
               />
+              <div>
+                <ContentSwitcher
+                  onChange={({ name }) => {
+                    name === 'one' ? setSelIndex(0) : setSelIndex(1);
+                  }}
+                  selectedIndex={selIndex}
+                  selectionMode="automatic"
+                >
+                  <Switch name="one" text="First section" />
+                  <Switch name="two" text="Second section" />
+                </ContentSwitcher>
+                {selIndex === 0 ? (
+                  <>
+                    <p>first section</p>
+                    {/* <TextInput /> */}
+                  </>
+                ) : (
+                  <>
+                    <p>second section</p>
+                    {/* <TextInput /> */}
+                  </>
+                )}
```

Comment:
We probably don't need to keep this in the example story if you were able to verify that it works as expected now.